src/types/TermControlUiaTextRange.cpp

```diff
@@ -129,16 +129,16 @@ void UiaTextRange::_TranslatePointToScreen(LPPOINT clientPoint) const
 
     // update based on TermControl padding
     auto padding = provider->GetPadding();
-    clientPoint->x += gsl::narrow<LONG>(padding.Left);
-    clientPoint->y += gsl::narrow<LONG>(padding.Top);
+    clientPoint->x += gsl::narrow<LONG>(padding.left);
+    clientPoint->y += gsl::narrow<LONG>(padding.top);
 }
 
-void UiaTextRange::_TranslatePointFromScreen(LPPOINT /*screenPoint*/) const
+void TermControlUiaTextRange::_TranslatePointFromScreen(LPPOINT /*screenPoint*/) const
 {
     // TODO GitHub #2103: NON-HWND IMPLEMENTATION OF SCREENTOCLIENT()
 }
 
-const COORD UiaTextRange::_getScreenFontSize() const
+const COORD TermControlUiaTextRange::_getScreenFontSize() const
 {
     // Do NOT get the font info from IRenderData. It is a dummy font info.
     // Instead, the font info is saved in the TermControl. So we have to

```

Comment:
since these are now in the same library and have been made to take another interface (IControlInfo), it seems like this could now be merged with UiaTextRangeBase (and have `Base` dropped).

@carlos-zamora has been talking about this for a while: we split them, but if we have a shared interface between them now maybe they can be merged again...

Reply:
I was thinking that too, but I wanted to get your team's input first. Especially since I now have conflicts because they keep changing 😅

Reply:
Yeah! Sorry about that! That's 100% my fault.

Reply:
@carlos-zamora do you think it makes sense to merge these into the base class? It will also take care of some static analysis issues at the same time.

Reply:
Yeah. Go for it.

Reply:
I looked into it more and the `screenInfoUiaProvider.cpp` files in `src/interactivity/win32` throw a wrench in this. I think I should hold off for now.

Reply:
Fair enough. Mind submitting an issue to track this? It'd be helpful to include what issue you ran into with `screenInfoUiaProvider.cpp`

Reply:
Sure, the biggest problem was that the `screenInfoUiaProvider.cpp` has things like proper parent navigation, which isn't in the base classes. It also had a `GetWindowHandle` method which doesn't have a good analogue in the base clases.